Source/WebKit/WebProcess/WebPage/WebPage.h

```diff
@@ -2220,6 +2220,10 @@ class WebPage : public API::ObjectImpl<API::Object::Type::BundlePage>, public IP
 
     RefPtr<WebCore::Page> m_page;
 
+#if PLATFORM(MAC)
+    bool m_renderServerMachExtensionConsumed { false };
```

Comment:
Where is this used?

Also, boolean variables should start with a prefix, maybe "was" in this case.

Reply:
Ah, that is a good point. Actually, it was only declared to make sure that the sandbox extension is consumed before `m_drawingArea` was initialized. Alternatively, we could move the initialization of `m_drawingArea` into the body. What do you think?

Thanks for reviewing?

Reply:
Another option is to consume the sandbox extension in the drawing area constructor.

Reply:
Either option would be better than the proposal in the PR I think.

Reply:
Yes, sounds good, I went for the last option :)